app/src/androidTest/java/org/mozilla/focus/helpers/MainActivityTestRule.kt

```diff
@@ -125,6 +127,10 @@ private fun hideFirstRun(appStore: AppStore) {
     runBlocking { job.join() }
 }
 
+private fun setNewOnboarding(enabled: Boolean) {
+    InstrumentationRegistry.getInstrumentation().targetContext.settings.isNewOnboardingEnable = enabled
```

Comment:
you could import the appContext variable from TestHelper

Reply:
Done !